pandas/io/formats/format.py

```diff
@@ -1744,7 +1742,7 @@ def format_percentiles(
     return [i + "%" for i in out]
 
 
-def is_dates_only(values: np.ndarray | DatetimeArray | Index | DatetimeIndex) -> bool:
+def is_dates_only_(values: np.ndarray | DatetimeArray | Index | DatetimeIndex) -> bool:
```

Comment:
ah sorry, I meant to rename the argument `get_format_datetime64` (else this function will need renaming in the other file where it's used)

Reply:
Oh- got it- let me just swap that around